src/reference/asciidoc/whats-new.adoc

```diff
@@ -25,6 +25,10 @@ The `spring.integration.channels.error.requireSubscribers=true` global property
 The `spring.integration.channels.error.ignoreFailures=true` global property is added to indicate that the global default `errorChannel` must ignore (or not) dispatching errors and pass the message to the next handler.
 See <<./configuration.adoc#global-properties,Global Properties>> for more information.
 
+An `AbstractPollingEndpoint` (source polling channel adapter and polling consumer) treats a `maxMessagesPerPoll == 0` as to skip calling the source.
+It can be changed to different value later on, e.g. via Control Bus.
```

Comment:
```suggestion
An `AbstractPollingEndpoint` (source polling channel adapter and polling consumer) treats `maxMessagesPerPoll == 0` as to skip calling the source.
It can be changed to different value later on, e.g. via a Control Bus.
```